### ISSUE TITLE
Also having a black screen

### ISSUE THREAD

**Issue by jimmy9065** [rien/reStream#96]
I have checked all the previous issues but couldn't find a solution for me. 
I'm using remarkable2 (ver 3.5.2.1807)
this is the log I have
```
[SSH] true
[SSH] cat /sys/devices/soc0/machine
[SSH] [ -f /dev/shm/swtfb.01 ]
[SSH] [ ! -f ~/restream ] && [ ! -f /opt/bin/restream ]
[SSH] PATH="$PATH:/opt/bin/:." restream -h 1404 -w 1872 -b 1 -f :mem:
ffplay version n6.0 Copyright (c) 2003-2023 the FFmpeg developers
  built with gcc 13.1.1 (GCC) 20230429
  configuration: --prefix=/usr --disable-debug --disable-static --disable-stripping --enable-amf --enable-avisynth --enable-cuda-llvm --enable-lto --enable-fontconfig --enable-gmp --enable-gnutls --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libdav1d --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libgsm --enable-libiec61883 --enable-libjack --enable-libjxl --enable-libmfx --enable-libmodplug --enable-libmp3lame --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librav1e --enable-librsvg --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-libzimg --enable-nvdec --enable-nvenc --enable-opencl --enable-opengl --enable-shared --enable-version3 --enable-vulkan
  libavutil      58.  2.100 / 58.  2.100
  libavcodec     60.  3.100 / 60.  3.100
  libavformat    60.  3.100 / 60.  3.100
  libavdevice    60.  1.100 / 60.  1.100
  libavfilter     9.  3.100 /  9.  3.100
  libswscale      7.  1.100 /  7.  1.100
  libswresample   4. 10.100 /  4. 10.100
  libpostproc    57.  1.100 / 57.  1.100
Input #0, rawvideo, from 'fd:':    0KB vq=    0KB sq=    0B f=0/0   
  Duration: N/A, start: 0.000000, bitrate: 525657 kb/s
  Stream #0:0: Video: rawvideo (Y800 / 0x30303859), gray, 1872x1404, 525657 kb/s, 25 tbr, 25 tbn
^C[rawvideo @ 0x7feeac000c80] Packet corrupt (stream = 0, dts = 210).
[rawvideo @ 0x7feeac295b40] Invalid buffer size, packet size 1707648 < expected frame_size 2628288
[SSH] kill $(pidof restream)
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
[SSH] kill $(pidof restream)
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
```

I can ssh into the remarkable and I check the `/dev/shm/` there is no file there. I thought it suppose to have a `/dev/shm/swtfb.01` according to #86, right?

I also did `ps |grep restream` and this is what I got
```
 1173 root     11172 R    restream -h 1404 -w 1872 -b 1 -f :mem:
```
which does make sense since I don't have the swtfb.01 file and this is suppose to happen according to the script.

wondering what I missed in the instruction. any idea?

**Comment by Eeems** [comment]
`/dev/shm/swtfb.01` is only available if you have [rm2fb](https://github.com/ddvk/remarkable2-framebuffer) installed. rm2fb does not support 3.5.2.1807 currently.

**Comment by jimmy9065** [comment]
thanks for the prompt reply @Eeems 
1. so do we have to use rm2fb to use reStream? ( in the script it does have logic to handle the case when swftb.01 is missing, but not sure if it suppose to work)
2. also, assume rm2fb is working, do I need to install that separately? (didn't see this step in the readme so I thought it comes with restream maybe :joy: )

**Comment by Eeems** [comment]
> thanks for the prompt reply @Eeems 
> 1. so do we have to use rm2fb to use reStream? ( in the script it does have logic to handle the case when swftb.01 is missing, but not sure if it suppose to work)
> 2. also, assume rm2fb is working, do I need to install that separately? (didn't see this step in the readme so I thought it comes with restream maybe :joy: )

If rm2fb is not installed or running, the file does not exist, so it instead uses the other settings.

To use rm2fb, you would install that separately, and only if it supports your OS version.

**Comment by jimmy9065** [comment]
thanks for the explanation! last question, 
do you know if "other settings" would work for reStream?
I mean I'm still seeing black screen so it either means that "other settings" is not working for me or "other settings" doesn't work at all for reStream.

**Comment by Eeems** [comment]
> thanks for the explanation! last question, do you know if "other settings" would work for reStream? I mean I'm still seeing black screen so it either means that "other settings" is not working for me or "other settings" doesn't work at all for reStream.

Since you have a black screen, no, they are not working. See #95 for some conversation on possible different settings to use.

**Comment by itay-grudev** [comment]
To address the dark image issue, add an `eq=gamma=10` filter, like so:

```
video_filters="$video_filters,transpose=3,eq=gamma=10"
```

**Comment by rien** [comment]
This should be fixed in the latest release by #100. Let me know if you encounter any additional issues.
